src/components/Profile/components/PersonalInfoList/index.js

```diff
@@ -183,6 +184,13 @@ const PersonalInfoList = ({
     />
   ) : null;
 
+  const cliftonStrengths = CliftonStrengths?.Strengths ? (
+    <ProfileInfoListItem
+      title="Clifton Strengths:"
+      contentText={CliftonStrengths.Strengths.join(', ')}
+    />
+  ) : '';
```

Comment:
I'm not sure that it matters, but usually `null` is a more common way to represent a non-existent element than the empty string.
```suggestion
  ) : null;
```